### ISSUE TITLE
Practice Problems points didn't get updated

### ISSUE THREAD

**Issue by diputsm41** [appirio-tech/arena-web#831]
I submitted correct answers for more than 3 problems in Practice Problems. After submitting, I always got the points (popup), but when I switched to list view, I don't see my points.

![mypoints](https://user-images.githubusercontent.com/57287585/68072050-e2597e80-fdb3-11e9-9214-ab175fe91e6f.png)

**Comment by kierenholt** [comment]
I'm having the same issue

**Comment by rocky1991** [comment]
I have the same issue too. Please fix it

**Comment by gurdevrana** [comment]
same with me. Points are not getting updated

**Comment by dan323** [comment]
Me too

**Comment by vindulanb** [comment]
Me too

**Comment by ergauravsoni** [comment]
Same issue

**Comment by amir-reza85** [comment]
Yes I solve aqaasadi and gcd (something like that) and showed in unsolved

I run sys tests and it write passed sys test and again dont show in solved

and just 2 pages of practice problems in very less

I cant find aqaasadi problems of SRM 787

**Comment by gurdevrana** [comment]
Are they going to solve this issue ?? as it is pending since 7-8 months...I think it's the major issue.
